tests/test_train.py

```diff
@@ -1,3 +1,106 @@
 import pytest,fastai
+from utils.fakes import *
+from utils.text import *
+from utils.param import *
+#from fastai.vision import *
 
-#I'm all empty, please fill me with tests about the functions in train.py
\ No newline at end of file
+## filename: test_train.py
+## tests functions in train.py
+
+## Class Learner, see documentation:  https://docs.fast.ai/train.html
+ 
+## Model fitting methods
+
+## run: pytest tests/test_train.py (add -s for screenoutput )
+
+
+@pytest.fixture(scope="module")
+def learn():
+    ## fixture for same model in fit and fit_one_cycle and demonstrate difference in behavior
+    ## test_fit and test_fit_one_cycle assert correctly with decommented models, 
+    ## fake_learner defaulted for performance reasons
+    
+    learn = fake_learner(50,50)
+
+    #path = untar_data(URLs.MNIST_TINY)
```

Comment:
This is a lot of commented code that we don't need.

Reply:
eliminated